README.md

```diff
@@ -28,21 +28,28 @@ Generally, you'll want to subclass `Exporter` and provide two required changes,
 
 You'll also need to specify a type for your headers to run over. We recommend you go with something like a `NamedTuple`, but any blob of data that is meaningful to you will do.
 
-For example, we'll start with a simple type:
+For example, we'll start by creating a file in for example `project/common/exports/llama_exporter.py`:
 
 ```python
-    Llama = NamedTuple('Llama', [('name': str), ('fluff_factor': int)])
-```
+from typing import List
+from csv_wrangler.exporter import Exporter, Header
+from project.llamas.models import Llama
 
-Now, we'll need our derived class:
+# We start by defining Llama's type
+Llama = NamedTuple('Llama', [('first_name': str), ('last_name': str), ('fluff_factor': int)])
 
-```python
+# We can define a helper function if needed
+def get_full_name(llama):
+    return "{} {}".format(llama.first_name, llama.last_name)
+
+
+# And we'll need our derived class
 class LlamaExporter(Exporter):
 
     headers = [
-        Header(label='name', callback=lambda llama: llama.name,
+        Header(label='name', callback=lambda llama: self.get_full_name(llama),
```

Comment:
This won't be on Self, as it's a free function

```Header(label='name', callback=lambda llama: get_full_name(llama),```

which can be generalized to

```Header(label='name', callback=get_full_name),```

Reply:
aah - nice